exonum-java-binding/core-its/src/test/java/com/exonum/binding/runtime/Pf4jServiceLoaderIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.runtime;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Test;
+import org.pf4j.DefaultPluginManager;
+import org.pf4j.PluginManager;
+
+class Pf4jServiceLoaderIntegrationTest {
+
+  @Test
+  void loadsFakeService() throws ServiceLoadingException {
+    String fakeServicePath = System.getProperty("exonum.fakeServicePath");
```

Comment:
does it work for running the test from Idea directly?

Reply:
Yes, as core ITs, if it is configured to use Maven configurations.